content/01.abstract.md

```diff
@@ -11,3 +11,20 @@ Publication with Manubot is continuous:
 When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
 Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers (e.g. DOIs, PubMed IDs, ISBNs, URLs), whose metadata is then retrieved and converted to a user-specified style.
 Manubot modernizes publishing to align with the ideals of open science by making it transparent, reproducible, immediate, versioned, collaborative, and free of charge.
+
+## Author summary
+
+Traditionally, scholarly manuscripts have been written in private by a predefined team of collaborators.
+But now the internet enables realtime open science, where project communication occurs online in a public venue and anyone is able to contribute.
+Dispersed teams of online contributors require new tools to jointly prepare manuscripts.
+
+Existing tools fail to scale beyond tens of authors and struggle to support iterative refinement of proposed changes.
+Therefore, we created a system for writing manuscripts based on collaborative version control, called Manubot.
```

Comment:
How about?
> Therefore, we created a system called Manubot for writing manuscripts based on collaborative version control.

Reply:
Done in https://github.com/greenelab/meta-review/pull/220/commits/0cfaf7794c4abf6e1b6bf9efc6dbe21712973369